docs-src/concepts/what-is-persistence.md

```diff
@@ -0,0 +1,46 @@
+---
+id: what-is-persistence
+title: What is Persistence?
+sidebar_label: Persistence
+description: Temporal Persistence store is a database used by Temporal Services to persist events generated and processed in the Temporal Cluster and SDK.
+tags:
+  - term
+  - explanation
+---
+
+Temporal Persistence store is a database used by [Temporal Services](/concepts/what-is-the-temporal-server) to persist events generated and processed in your Temporal Cluster and SDK.
+
+A Temporal Cluster's only required dependency for basic operation is the Persistence database.
+Multiple types of databases are supported.
+
+![Persistence](/diagrams/temporal-database.svg)
+
+The database stores the following types of data:
+
+- Tasks: Tasks to be dispatched.
+- State of Workflow Executions:
+  - Execution table: A capture of the mutable state of Workflow Executions.
+  - History table: An append only log of Workflow Execution History Events.
+- Namespace metadata: Metadata of each Namespace in the Cluster.
+- [Visibility](/concepts/what-is-visibility) data: Enables operations like "show all running Workflow Executions".
+  For production environments, we recommend using Elasticsearch as your Visibility store.
+
+An Elasticsearch database must be added to enable [Advanced Visibility](/concepts/what-is-advanced-visibility) on Temporal Server versions 1.19.1 and earlier.
```

Comment:
"Elasticsearch database", maybe just "Connection info to Elasticsearch must be set to use advanced visibility....."